README.md

```diff
@@ -256,8 +256,7 @@ The object structure is as follows:
 }
 ```
 #### { src; dest } Object Rules
- - if `src` is a non-glob path to a single file, then `dest` should include the filename and extension. For example:
- `{ src: "lib/Promise/promise.brs", dest: "source/promise.brs"}`
+ - if `src` is required to follow glob style (i.e. forward-slashes only). You can learn more [here](https://www.npmjs.com/package/fast-glob?activeTab=readme#how-to-write-patterns-on-windows).
```

Comment:
I think the original message was actually correct, and gets lost with the new message. The point of this comment is "if you reference 1 file in src, please reference 1 file in dest". 

We should add a 
```suggestion
- if `src` is a non-glob path to a single file, then `dest` should include the filename and extension. For example:
```